zerver/actions/message_edit.py

```diff
@@ -824,7 +824,17 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
 
     send_event(user_profile.realm, event, users_to_be_notified)
 
-    if len(changed_messages) > 0 and new_stream is not None and stream_being_edited is not None:
+    changed_topic = False
```

Comment:
I think the name `is_topic_changed` is better for this variable since I thought `changed_topic` contains the name of the changed topic

Reply:
That's a good point, i will change that